client/me/billing-history/main.jsx

```diff
@@ -21,27 +21,23 @@ import Main from 'components/main';
 import DocumentHead from 'components/data/document-head';
 import PageViewTracker from 'lib/analytics/page-view-tracker';
 import QueryBillingTransactions from 'components/data/query-billing-transactions';
-import { purchasesRoot } from 'me/purchases/paths';
-import { getPastBillingTransactions, getUpcomingBillingTransactions } from 'state/selectors';
+import { getPastBillingTransactions } from 'state/selectors';
 
-const BillingHistory = ( { pastTransactions, upcomingTransactions, translate } ) => (
+const BillingHistory = ( { pastTransactions, translate } ) => (
 	<Main className="billing-history">
 		<DocumentHead title={ translate( 'Billing History' ) } />
 		<PageViewTracker path="/me/purchases/billing" title="Me > Billing History" />
 		<MeSidebarNavigation />
 		<QueryBillingTransactions />
 		<PurchasesHeader section={ 'billing' } />
 		<Card className="billing-history__receipts">
-			<BillingHistoryTable transactions={ pastTransactions } />
-		</Card>
-		<Card href={ purchasesRoot }>
```

Comment:
This is removing the following card:

![](https://cldup.com/0xtk40kGyo.png)

I don't think it was very useful anyway, but just wanted to make sure that this removal is intentional.